templates/email/kingston/_bulky_data.html

```diff
@@ -0,0 +1,9 @@
+[%
+
+SET payment = report.get_extra_field_value('Payment_Details_Payment_Amount');
+SET collection_date = cobrand.bulky_nice_collection_date(report);
+SET report_date = cobrand.bulky_nice_collection_date(report.confirmed);
```

Comment:
I think this function should only be used on the report/collection date string, not a DateTime from the database, - use strftime directly otherwise I think.

Reply:
This doesn't appear to have changed, nor in Sutton.

Reply:
Was it meant to be removed now default exists?